src/components/Button/Button.scss

```diff
@@ -7,37 +7,37 @@
   align-items: center;
```

Comment:
Forgot to do this initially, but since you're in here, please add a `.cdc-react` class to this.

Reply:
Updated :)